chickadee/io.py

```diff
@@ -49,10 +49,230 @@
     data_type="positiveInteger",
 )
 
-end_date = LiteralInput(
-    "end_date",
-    "End Date",
-    abstract="Defines the end of the calibration period",
-    default=date(2005, 12, 31),
-    data_type="date",
-)
+
+general_options_input = [
+    LiteralInput(
+        "units_bool",
+        "Units Boolean",
+        abstract="Whether to check the input units and convertthem to the target output units",
```

Comment:
Just one minor type "convertthem" needs a space. You could also be more concise with:
"Check and convert input units to output units". Do you think it's still clear without "Whether"? To me it is.